lib/bkblz/v1/request.rb

```diff
@@ -39,7 +39,6 @@ def url_suffix(suffix=nil)
 
       def send(session)
         request = build_request session
-        request.set_content_type 'application/x-www-form-urlencoded'
```

Comment:
didn't you just add this line in the pull request?

could you rebase this pull request to squash these changes together so I can see the final result more easily.